src/ServiceManagement/Compute/Compute.Tests/Compute.Tests.csproj

```diff
@@ -172,11 +173,38 @@
       <Name>ComputeManagement</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.Threading.Tasks, Version=1.0.12.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
This dependency should already include all the files you need: https://github.com/Azure/azure-sdk-for-net/pull/2957/files#diff-27bb0edb25bb7f42bd18d1985d5ff50bR167 .  This may be the cause of build failures.